Dockerfiles/agent-ot/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG BASE_IMAGE_DD_VERSION=latest
+FROM docker.io/datadog/agent:$BASE_IMAGE_DD_VERSION AS baseimage
+
+ARG CIBUILD
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+FROM baseimage
+
+# copy otel requirements
+COPY otel-agent /opt/datadog-agent/embedded/bin/
+COPY otel-config.yaml /etc/datadog-agent/otel-config.yaml
+
+COPY s6-services/otel /etc/services.d/otel
+
+
+# Update entrypoints and what not
+COPY entrypoint.sh /bin/entrypoint.sh
+COPY entrypoint.d /opt/entrypoints
+RUN chmod 755 /bin/entrypoint.sh \
```

Comment:
nit: Isn't most of of the directories already created from parent and only changed files would require changes?

Reply:
I figured that since we were overriding the base image with the `HEAD` entrypoints and services I might as well just re-run the entire thing. Figured it wouldn't hurt to be redundant.